modules/ossm-about-concepts-kiali.adoc

```diff
@@ -0,0 +1,71 @@
+// Module included in the following assemblies:
+// about/ossm-about-concepts-assembly.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="ossm-about-concepts-kiali_{context}"]
+= {SMProductName} and Kiali
+
+//Kiali attributes are in the works, and possible {KialiProduct} will refer to Kiali provided by Red Hat, and not Kiali Operator provided by Red Hat.
+//Kiali attributes are outside the scope of this PR.
+//TP1 content. IA influx, likely everything will change for GA.
+
+Kiali is based on the open source link:https://kiali.io/[Kiali project]. Kiali provided by Red Hat is composed of three parts:
+
+* {KialiProduct}
+* Kiali Server
+* {SMPlugin}
+
+Working together, they form the user interface (UI) for {SMProduct}. Kiali provides visibility into your service mesh by showing you the microservices and how they are connected.
+
+Kiali provides observability into the Service Mesh running on {ocp-product-title}. Kiali helps you define, validate, and observe your Istio service mesh. It helps you to understand the structure of your service mesh by inferring the topology, and also provides information about the health of your service mesh.
+
+Kiali provides an interactive graph view of your mesh namespaces in near real time that provides visibility into features like circuit breakers, request rates, latency, and even graphs of traffic flows. Kiali offers insights about components at different levels, from Applications to Services and Workloads, and can display the interactions with contextual information and charts on the selected graph node or edge.
+
+Kiali also provides the ability to validate your Istio configurations, such as gateways, destination rules, virtual services, mesh policies, and more. Kiali provides detailed metrics, and a basic Grafana integration is available for advanced queries. Distributed tracing is provided by integrating {TempoName} and {OTELName} into the Kiali console.
+
+[id="kiali-architecture_{context}"]
+== Kiali architecture
+
+Kiali Server (back end):: This component runs in the container application platform and communicates with the service mesh components, retrieves and processes data, and exposes this data to the console. The Kiali Server does not need storage. When deploying the Server to a cluster, configurations are set in ConfigMaps and secrets.
+
+Kiali console (front end):: The Kiali console is a web application. 	The Kiali Server serves the Kiali console, which then queries back to the Kiali Server for data to present it to the user.
```

Comment:
```suggestion
Kiali console (front end):: The Kiali console is a web application. The Kiali Server serves the Kiali console, which then queries back to the Kiali Server for data to present it to the user.
```

Reply:
Where this is a definition for the console, can you make this sentence console-centric?

```suggestion
Kiali console (front end):: The Kiali console is a web application. The console queries the Kiali Server for data to present it to the user.
```